extensions/sentencepiece/build.gradle

```diff
@@ -62,7 +62,8 @@ apply from: file("${rootProject.projectDir}/tools/gradle/cpp-formatter.gradle")
 task compileJNI {
     doFirst {
         if (System.properties['os.name'].toLowerCase(Locale.ROOT).contains("mac")
-                || System.properties['os.name'].toLowerCase(Locale.ROOT).contains("linux")) {
+                || System.properties['os.name'].toLowerCase(Locale.ROOT).contains("linux")
+                || System.properties['os.name'].toLowerCase(Locale.ROOT).contains("windows")) {
```

Comment:
I don't think this work for windows, you need specify a full "git-bash.exe" path. see: https://stackoverflow.com/questions/57614140/execute-git-bash-from-gradle-task


or you can create a "build.cmd" file, see: https://github.com/deepjavalibrary/djl/blob/master/engines/pytorch/pytorch-native/build.gradle#L31-L33